providers/softlayer/src/test/java/org/jclouds/softlayer/features/VirtualGuestApiLiveTest.java

```diff
@@ -132,9 +133,13 @@ public void testSetTagsOnVirtualGuest() throws Exception {
       VirtualGuest found = virtualGuestApi.getVirtualGuest(virtualGuest.getId());
       Set<TagReference> tagReferences = found.getTagReferences();
       assertNotNull(tagReferences);
-      for (String tag : tags) {
-         Iterables.contains(tagReferences, tag);
+
+      ImmutableSet.Builder<String> actualTagsBuilder = ImmutableSet.builder();
+      for (TagReference ref : tagReferences) {
+         actualTagsBuilder.add(ref.getTag().getName());
       }
+      Set<String> actualTags = actualTagsBuilder.build();
+      assertThat(actualTags).containsAll(tags);
```

Comment:
Guava 20 adds a `CheckReturnValue` annotation to several methods which we must consume when compiling with the newer library.